app/Providers/AppServiceProvider.php

```diff
@@ -23,6 +24,13 @@ public function boot()
      */
     public function register()
     {
-        //
+        $this->app->bind('Abraham\TwitterOAuth\TwitterOAuth', function () {
+            return new TwitterOAuth(
+                env('TWITTER_CONSUMER_KEY'),
```

Comment:
If these are being introduced here, could we consider moving them to config vars that read the env() files? If not, I still want to do that but we can do it later if need be.